sdk/azurestackhci/Microsoft.Azure.Management.AzureStackHCI/src/Microsoft.Azure.Management.AzureStackHCI.csproj

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <!-- Please do not move/edit code below this line -->
+  <Import Project="$(RepoEngPath)/mgmt/AzSdk.reference.props" />
+  <!-- Please do not move/edit code below this line -->
+
+  <PropertyGroup>
+    <Description>Microsoft Azure Stack HCI Management Library</Description>
+    <Version>1.0.0</Version>
```

Comment:
We usually use `1.0.0-preview.1` as the version of the first release. Then `1.0.0-preview.2`, `1.0.0-preview.3` ...
When we are sure it's stable, change it to `1.0.0`

Read more about versioning guide: https://github.com/Azure/azure-sdk-for-net/blob/master/doc/dev/Versioning.md

Reply:
Fixed, thanks!